server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/delivery/SieveExecutor.java

```diff
@@ -121,10 +120,7 @@ public boolean execute(MailAddress recipient, Mail mail) throws MessagingExcepti
         Preconditions.checkNotNull(recipient, "Recipient for mail to be spooled cannot be null.");
         Preconditions.checkNotNull(mail.getMessage(), "Mail message to be spooled cannot be null.");
         boolean isSieveNotification = AttributeUtils.getValueAndCastFromMail(mail, SIEVE_NOTIFICATION, Boolean.class).orElse(false);
-        if (isSieveNotification) {
-           	return false;
-        }
-        return sieveMessage(recipient, mail);
+        return !isSieveNotification ? sieveMessage(recipient, mail) : false;
```

Comment:
The previous version was more explicit.

`4 spaces` indent is all of what is needed to fix the checkstyle, no?

Reply:
I agree, we tend to not use the ternary operation in the code base.